test/regression/builder/road_network_builder_test.cc

```diff
@@ -587,7 +587,7 @@ std::vector<DirectionUsageTruthTable> InstantiateDirectionUsageBuilderParameters
            {LaneId("1_2_3"), SRange(0., 33.4), "Bidirectional", strict_severity},
            {LaneId("1_2_2"), SRange(0., 33.4), "Bidirectional", strict_severity},
            {LaneId("1_2_1"), SRange(0., 33.4), "AgainstS", strict_severity},
-           {LaneId("1_2_-1"), SRange(0., 33.4), "Undefined", strict_severity},
+           {LaneId("1_2_-1"), SRange(0., 33.4), "WithS", strict_severity},
            {LaneId("1_2_-2"), SRange(0., 33.4), "Bidirectional", strict_severity},
            {LaneId("1_2_-3"), SRange(0., 33.4), "Bidirectional", strict_severity},
            {LaneId("1_2_-4"), SRange(0., 33.4), "Bidirectional", strict_severity},
```

Comment:
Create an extra integration test with a new xodr:
 - Road with RHT
   - Lanes negative and positive
   - Lanes with lane direction: (None) / Standard / Reverse / Both
   - Lanes with user data
 - Road with LHT
   - Lanes negative and positive
   - Lanes with lane direction: (None) / Standard / Reverse / Both
   - Lanes with user data
 - Road without rule set.
   - Lanes negative and positive
   - Lanes with lane direction: (None) / Standard / Reverse / Both
   - Lanes with user data
   
   
Add a case in this battery of test for testing all the cases.

Reply:
Done.